dot/digest/block_import.go

```diff
@@ -53,7 +53,7 @@ func (h *BlockImportHandler) handleDigests(header *types.Header) error {
 		digest := consensusDigests[i]
 		err := h.handleConsensusDigest(&digest, header)
 		if err != nil {
-			logger.Errorf("cannot handle consensus digest: %w", err)
+			return err
```

Comment:
Maybe add an error message here? `handleConsensusDigest` doesn't have it either in couple of places